scripts/tools/silabs/ota/ota_multi_image_tool.py

```diff
@@ -319,6 +350,15 @@ def any_base_int(s): return int(s, 0)
     create_parser.add_argument('--app-build-date', type=str,
                                help='Application build date (string)')
 
+    create_parser.add_argument('-wifi', "--wifi_ta_input_file",
```

Comment:
this short param is not relevantly named -wifi for the TA image file. I suggest using something else
`-tai `maybe

the following args could then follow that pattern
```
-tav
-tavs
-tad
```
etc

Reply:
ok I have incorporated `-tai`  only and others are inline with the existing patterns.